.github/workflows/build-simd-image-from-tag.yml

```diff
@@ -25,6 +25,11 @@ jobs:
            with:
             ref: "${{ env.GIT_TAG }}"
             fetch-depth: 0
+        - uses: actions/setup-python@v5
+           with:
+             python-version: '3.10'
+        - name: Install dependencies
```

Comment:
I guess this is necessary to use the script to get the libwasm version and checksum